### DISCUSSION TITLE
Einfache Konfiguration für PV Überschuss

### DISCUSSION THREAD

**Discussion by ck873** [cathiele/homeassistant-goecharger#99] [Q&A]
Ich habe alles eingerichtet und sehe alle Parameter meines go-e Chargers (Firmware v042) in Home Assistant. Ich kann diesen auch über den Switch im Dashboard starten und stoppen. Allerdings führt eine Änderung der "Max current" Auswahl (z.B. "8" ) zu keiner Änderung im go-e Charger. Somit verstehe ich nicht wie ich eine PV Überschuss Steuerung realisieren soll? Meine Einstellungen sind in der configuration.yaml und automation.yaml direkt aus der Beschreibung kopiert ("111111" habe ich angepasst). Was mache ich falsch oder was fehlt noch damit 'input_select.goecharger_max_current' zu einer Änderung am go-e Charger führt?

**Comment by AchimQ** [comment]
Deine Diskussion ist schon ein paar Tage alt, aber ich beschäftige mich erst jetzt mit dem Thema.
Hast Du die beiden Automations aus der Dokumentation in die Konfiguration von Homeassistant aufgenommen? Die eine schaltet den Charger nach der UI Auswahl und die andere setzt die UI Auswahl, wenn der Charger den Wert ändert, z.B. über die App.
